tests/storage/restore.go

```diff
@@ -393,11 +392,7 @@ var _ = SIGDescribe("VirtualMachineRestore Tests", func() {
 			})
 
 			It("[test_id:5257]should reject restore if VM running", func() {
-				patch, err := patch.New(patch.WithReplace("/spec/runStrategy", v1.RunStrategyAlways)).GeneratePayload()
-				Expect(err).ToNot(HaveOccurred())
-
-				vm, err := virtClient.VirtualMachine(vm.Namespace).Patch(context.Background(), vm.Name, types.JSONPatchType, patch, metav1.PatchOptions{})
-				Expect(err).ToNot(HaveOccurred())
+				Expect(virtClient.VirtualMachine(vm.Namespace).Start(context.Background(), vm.Name, &v1.StartOptions{})).To(Succeed())
```

Comment:
Do you think we can create the VM with RunStrategyAlways in the first place?

Reply:
No, the VM is created in the BeforeEach() and the other tests don't expect it to be running.

Reply:
IMO  We should extract this test from this scope or move the creation to the tests.

Reply:
Good idea, but for another PR.

Reply:
IMO this is in the scope of the current PR.